contracts/sol6/Dao/KyberDAO.sol

```diff
@@ -770,7 +770,7 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5 {
                 // so network fee should be less than 50%
                 require(
                     options[i] < BPS / 2,
-                    "validateParams: Fee campaign option value is too high"
+                    "validateParams: fee campaign option value is high"
```

Comment:
lets make this clearer.


validateParams: network fee must be smaller then BPS / 2